core/templates/pages/topic-viewer-page/topic-header/topic-header.component.html

```
@@ -0,0 +1,26 @@
+<div class="topic-header-breadcrumbs">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)